### ISSUE TITLE
Slimeling loses sitting status and follows owner after server restarts

### ISSUE THREAD

**Issue by database64128** [micdoodle8/Galacticraft#3813]
Minecraft version: 1.12.2
Forge 2847
Galacticraft version: 236 and/or 237

SMP Server with Sponge 2825 7.1.6

Any Galacticraft add-ons installed? please list all
Extra Planets 0.5.8

Starting from Galacticraft 236, when you restart the server, slimelings will lose sitting status and follow owner. I'm now on 237 and still have this issue.

**Comment by database64128** [comment]
Any plan to fix it?

**Comment by EzerArch** [comment]
Does the same bug happen with tamed wolves? I guess the slimeling uses the same wolf code.

**Comment by database64128** [comment]
No. My tamed wolves are in the same chunk with the slimelings, and they remain sitting when the chunk is reloaded.
